docs/source/scores/guidelines.rst

```diff
@@ -5,8 +5,62 @@ DCML Score Conventions
 .. contents:: Contents
    :local:
 
+General Principles
+==================
+
+These guidelines are intended to ensure that the scores produced by and for the DCML are consistent and high-quality
+encodings of some print or manuscript edition. Although they represent digital editions, first and foremost they
+are to be considered as *research data*. As such, we neglect details of layout in favour of rigorous conventions
+regarding the semantically correct encoding of score elements. Semantic, in this context, means that wherever
+possible the various score elements are to be created with the corresponding tools and features that the
+typesetting software MuseScore provides. For example, a ``cresc.`` performance indication must not be encoded
+as a "normal" staff text: It has the semantics of a dynamic marking and is available as such in the "Lines" palette.
+A large part of these guidelines is concerned with stipulating how various score elements are to be encoded.
+
```

Comment:
We should also emphasize that, unless otherwise indicated, DCML scores are produced with a "fitness to use" mindset. While we prefer to rely on authoritative sources and scholarly editions, we do not aspire to produce Urtext scores or reference-quality material. Rather, we provide scores that are "good enough" for the research questions that motivated their production. These are typically computational-musicology questions.